### PR TITLE
Update linking.yaml

### PR DESCRIPTION
**Pull request by kengie**
Adding JC and Iowa City BHR people, commenting out old Middlesex and Dane entity sets.

### CODE REVIEW

**Review by katherinebernstein — APPROVED** [review]

File: src/main/resources/linking.yaml [hunk 1]
```diff
@@ -19,39 +19,41 @@ blacklist:
   - 2ef9d5ce-0245-4f85-95ab-6e4b06f8c8a7
   - 71ddddd6-3c5d-4a8b-8fe3-808d1eb13f19
 whitelist:
-  - d189f747-fa39-4b41-b86e-a056100ed2fa # JC3
```
Comment (makosblade) [inline src/main/resources/linking.yaml:22]: Just want to flag that the whitelist is not used by the Background linking process anymore

**Review by makosblade — APPROVED** [review]

File: src/main/resources/linking.yaml (hunk 1 above)
Comment (makosblade) [reply src/main/resources/linking.yaml:22]: Neither is the blacklist btw

**Comment by makosblade** [comment]
I should clarify, neither the whitelist nor the blacklist is used in the linking process anymore

**Comment by makosblade** [comment]
Uhhh this should not be used

**Pull request merged**